.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/
+.idea/
```

Comment:
we should add `.DS_Store` too, maybe I can do it later

Reply:
Added